owlkettle.nimble

```diff
@@ -18,7 +18,7 @@ task examples, "Build examples":
   withDir "examples":
     for file in findExamples("."):
       echo "INFO: Compile " & file
-      exec "nim c --hints:off --verbosity:0 " & file
+      exec "nim c --hints:off --path:.. --verbosity:0 " & file
```

Comment:
Why is this required?

Reply:
I currently very intentionally have no global version of owlkettle installed to make sure when compiling this it **has** to use the local version. If I don't provide a path to this local version (aka the owlkettle **in this repo** instead of the installed owlkettle version) I get:
```nim
/home/philipp/dev/owlkettle/examples/introduction.nim(23, 8) Error: cannot open file: owlkettle
```

Which indicates to me that if you run `nimble examples` as "testing" locally, it'll not use your local owlkettle version.
--path:.. allows you to specify that it shall add wherever your owlkettle repo is to the path, where it can find and thus import owlkettle.nim

This **should** guarantee that it uses the owlkettle files of your repository rather than of whatever owlkettle version you got installed.

Reply:
Ah, makes sense.